usage/NODES.md

```diff
@@ -10,3 +10,13 @@ Indices of partition indicate number of available nodes within a partition, e.g.
 | rome[1-7]    	|  96  		| 2:48:1 			| AMD EPYC 7642 	| N/A 			| 1000 		| Ubuntu 20.04 focal 		| 2.7.18, 3.8.10 | 
 | turing[1-2]  	|  64  		| 2:16:2 			| AMD EPYC 7302 	| 6x Tesla T4 (TU104GL) | 500  		| Ubuntu 20.04 focal 		| 2.7.18, 3.8.10 |
 | vmware[1-4]  	|   1  		|  1:1:1 			| Intel Xeon E5-2680 v4 | N/A 			| 1 		| Ubuntu 18.04.5 bionic 	| N/A | 
+
+
+## GPU compute
+The two nodes of the `turing` partition has different driver- and software-stacks as depicted below, this is to accomodate projects which require newer features. 
+Note that `turing02` might be unstable due to this newer stack, report any issues and requests to [falke](mailto:falkeboc@cs.aau.dk).
+
+| Node Name  | CUDA Driver  | CUDA Toolkit       |
+|------------|--------------|--------------------|
+| `turing01` | cuda-driver-450 | cuda-toolkit-11-0    |
+| `turing02` | cuda-driver-535 | cuda-toolkit-12.2 + cuda-toolkit-12.3 |
```

Comment:
There is a potential version mismatch here.
The driver version 535, supports up to 12.2, however the default cuda toolkit e.g. the symbolic link /usr/local/cuda points to the cuda-toolkit-12.3 which could cause locally compiled programs to fail when running.

Maybe the best choice is to delete cuda-toolkit-12.3 and setup the symlink to point to cuda-toolkit-12.2 instead

Reply:
I agree. Left both 12.2 and 12.3 because I was unsure which the GPU DBM project required. 

`cuda-toolkit-12.3` removed from turing02.